docs/frontend-system/architecture/03-extensions.md

```diff
@@ -83,7 +83,7 @@ Note that while the `createExtension` is public API and used in many places, it
 
 ## Extension Data
 
-Communication between extensions happens in one direction, from one child extension through the attachment point to its parent. The child extension outputs data which is then passed as inputs to the parent extension. This data is called Extension Data, where the shape of each individual piece of data is described by an Extension Data Reference. These references are created separately from the extensions themselves, and can be shared across multiple different kinds of extensions. Each reference consists of an ID and a TypeScript type that the data needs to conform to, and represents one type of data that can be shared between extensions.
+Communication between extensions happens in one direction, from one child extension through the attachment point to its parent. The child extension outputs data which is then passes as inputs to the parent extension. This data is called Extension Data, where the shape of each individual piece of data is described by an Extension Data Reference. These references are created separately from the extensions themselves, and can be shared across multiple different kinds of extensions. Each reference consists of an ID and a TypeScript type that the data needs to conform to, and represents one type of data that can be shared between extensions.
```

Comment:
I think this one was correct :pray:

Reply:
You're right, I misread it as "which it then passes". Will fix it.